infra/helmfile.yaml

```diff
@@ -0,0 +1,23 @@
+filepath: ""
```

Comment:
let's rename infra to jx-vault, I think we follow the convention where the folder name is the namespace

Reply:
Good idea